bin/rucio

```diff
@@ -676,26 +664,47 @@ def list_dids_extended(args):
     except InvalidObject:
         scope = args.did[0]
         name = '*'
+
     if scope not in client.list_scopes():
         logger.error('Scope not found')
         return FAILURE
 
     if args.recursive and '*' in name:
         logger.error('Option recursive cannot be used with wildcards')
         return FAILURE
-    elif ('name' in filters) and (name != '*'):
-        logger.error('You cannot use a wildcard query and a filter by name')
+    else:
+        if filters:
+            if ('name' in filters) and (name != '*'):
+                logger.error('Must have a wildcard in did name if filtering by name')
+                return FAILURE
+
+    try:
+        filters, type_ = parse_did_filter_from_string_fe(args.filter, name)
```

Comment:
What if `--filter` is not provided? It's optional

Reply:
`args.filter` defaults to None if not specified as an argument, it's then handled by the function